### DISCUSSION TITLE
See all goal event stats whilst filtering for one goal event

### DISCUSSION THREAD

**Discussion by NicholasJohnstone** [plausible/analytics#1637] [Anything Else]
When I filter by a specific completed goal, the dashboard doesn't show the filtered stats for any of the other goals. Is there any way to get this information?
I would really like to correlate events, e.g. if someone completed this event are they more likely to complete a different event.

**Comment by metmarkosaric** [comment]
thanks! this is not possible at this time unfortunately. we may be able to add it in the future. there's a similar idea here already https://github.com/plausible/analytics/discussions/846
